doc/guide/packages.xml

```diff
@@ -186,6 +186,39 @@ $ cockpit-bridge --packages
         <listitem><para>The relative path to the HTML file within the package that implements
           the menu item or tool.</para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term>keywords</term>
+        <listitem><para>Keywords that describe the page and which are used for searching.
+          These keywords should be lowercase. Page label is prepended as first keyword.
+          Keywords is object of keyword items - content of such item is described below.
```

Comment:
I guess the name is kinda pointless - it was introduced when I was also getting content of services page - where name was each service identifier. The idea was that it can update specific service - but in the end it was not used anyway.
I think it could be list  of objects, like:
```
"keywords": [                                                       
    {                                                    
        "matches": ["k1", "k2"]
    },                                                              
    {                                                   
        "matches": ["k2", "k3"],              
        "goto": "/network/firewall"                                 
    }   
] 
```

I know it is a bit complicated still for just specifying list of keywords, but with this it is possible to specify for example for each service its search terms (like service name and unit name) together with path and also specify that it should not be translated. And I did not want to have two different structures for manifest and for pages themselves.
Would this make more sense @martinpitt ?

Reply:
Yes, I think that's a good compromise. And it still leaves open the possibility of supporting  list entries which are plain strings, so in that case
```json
"keywords": [                                                       
    "k1",
    "k2",
    {                                                   
        "matches": ["k2", "k3"],              
        "goto": "/network/firewall"                                 
    }   
] 
```

But we can add that in a follow-up when needed, it doesn't break compatibility.